### PR TITLE
refactor(airflow): remove verbose log from airflow plugin

### PR DESCRIPTION
**Pull request by bskim45**
This PR removes the verbose log from the airflow plugin.
These logs are pretty verbose since `task_policy` are executed for all tasks in dags whenever the airflow CLI command is executed.

image

## Checklist

- [x] The PR conforms to DataHub's [Contributing Guideline](https://github.com/datahub-project/datahub/blob/master/docs/CONTRIBUTING.md) (particularly [Commit Message Format](https://github.com/datahub-project/datahub/blob/master/docs/CONTRIBUTING.md#commit-message-format))
- [ ] Links to related issues (if applicable)
- [ ] Tests for the changes have been added/updated (if applicable)
- [ ] Docs related to the changes have been added/updated (if applicable). If a new feature has been added a Usage Guide has been added for the same.
- [ ] For any breaking change/potential downtime/deprecation/big changes an entry has been made in [Updating DataHub](https://github.com/datahub-project/datahub/blob/master/docs/how/updating-datahub.md)

### CODE REVIEW

File: metadata-ingestion/src/datahub_provider/_plugin.py [hunk 1]
```diff
@@ -275,7 +275,6 @@ def custom_on_success_callback(context):
 
 
 def task_policy(task: BaseOperator) -> None:
-    print(f"Setting task policy for Dag: {task.dag_id} Task: {task.task_id}")
```
Comment (hsheth2) [inline metadata-ingestion/src/datahub_provider/_plugin.py:278]: maybe we can change it to `logger.debug`?

File: metadata-ingestion/src/datahub_provider/_plugin.py [hunk 2]
```diff
@@ -302,8 +301,6 @@ def _patch_policy(settings):
 
 
 def _patch_datahub_policy():
-    print("Patching datahub policy")
```
Comment (hsheth2) [inline metadata-ingestion/src/datahub_provider/_plugin.py:305]: imo this one should still be logger.info - I believe this one only gets printed once (as opposed to per dag per task), so a higher log level is ok

**Review by hsheth2 — COMMENTED** [review]
had some minor questions, but thanks for taking this on

File: metadata-ingestion/src/datahub_provider/_plugin.py (hunk 1 above)
Comment (bskim45) [reply metadata-ingestion/src/datahub_provider/_plugin.py:278]: 4e26016b2e0965fc390ae2760ad1f22eb78ef3a3

File: metadata-ingestion/src/datahub_provider/_plugin.py (hunk 2 above)
Comment (bskim45) [reply metadata-ingestion/src/datahub_provider/_plugin.py:305]: 4e26016b2e0965fc390ae2760ad1f22eb78ef3a3

**Review by hsheth2 — APPROVED** [review]
Lgtm, thanks!

**Pull request merged**